tedana/workflows/tedana.py

```diff
@@ -109,32 +104,28 @@ def _get_parser():
                               'logcosh (default), cube, exp'),
```

Comment:
While we're cleaning up the CLI, I'd say we should remove the option to pass different cost functions. This is an example we had originally listed in the Roadmap of something we thought was too complex for the non-power user (since power users can always update the cost function themselves)

Reply:
I thought the distinction between power and non-power users was the defaults, not the actual arguments. Am I misunderstanding that?

Still, I agree that the cost function is probably not something we need an argument for. I doubt it affects much.

Reply:
I think it's both, to an extent. Removing arguments implies a very strong default, whereas having them implies a weaker one. I think this is something we can have a very strong default on -- I've never needed to change the ICA cost function, for example. Not to say you couldn't need to, ever ! But it's reasonable as a strong default.

Reply:
That's fair. Dropping `cost` now.